website_portal_medical_prescription_order_line/controllers/main.py

```diff
@@ -28,6 +28,21 @@ def my_medical(self, **kw):
             ('patient_id.parent_id', 'child_of', [partner_id.id]),
         ])
 
+        url = str(request.httprequest)
```

Comment:
Here are some screenshots of the error (the return from `request.args.getlist('patients')`):

![screen shot 2017-04-03 at 9 19 03 am](https://cloud.githubusercontent.com/assets/18649466/24619542/302d0b7e-184f-11e7-8293-5dc789a7e35f.png)

![screen shot 2017-04-03 at 9 18 56 am](https://cloud.githubusercontent.com/assets/18649466/24619551/34ac9e62-184f-11e7-9333-00c77f9c0ea9.png)